packages/models/tests/server/organizations_test.coffee

```diff
@@ -32,3 +33,73 @@ describe 'Organization', ->
       expect(organization.validate("name")).not.to.be.ok
       organization.set('name', 'Test')
       expect(organization.validate("name")).to.be.ok
+
+  it 'can have members', (test, waitFor) ->
+    memberProfile = new UserProfile()
+    memberProfile.set(fullName: 'Snoopy')
+    id = memberProfile.save()
+    organization.save()
+    organization.addMember(id)
+    expect(
+      organization.getMemberProfiles().map((x)-> x.fullName)
+    ).to.include('Snoopy')
+
+  it 'can have admins', (test, waitFor) ->
+    memberProfile = new UserProfile()
+    memberProfile.set(fullName: 'TestUser')
+    memberProfile.save()
+    organization.set('name', 'TestOrg')
+    organization.save()
+    organization.addAdmin(memberProfile._id)
+    expect(
+      organization.getAdminProfiles().map((x)-> x.fullName)
+    ).to.include('TestUser')
```

Comment:
On a model level, what do you mean by "admins are also members?" Do you think there should be a unit test for a user who has the organization in adminOfOrgs but not memberOfOrgs?  What would you expect to be the result of userIsAdmin and userIsMember for that user?